client/state/ui/guided-tours/actions.js

```diff
@@ -21,14 +21,16 @@ import { getPreference } from 'state/preferences/selectors';
  * @param {Object} options Options object, see fn signature.
  * @return {Object} Action object
  */
-export function showGuidedTour( { shouldShow, shouldDelay = false, tour = 'main' } ) {
+export function showGuidedTour( { shouldShow, shouldDelay = false, tour } ) {
 	const showAction = {
 		type: GUIDED_TOUR_SHOW,
 		shouldShow,
 		shouldDelay,
 		tour,
 	};
 
+	// TODO(mcsf): track this *somewhere*, now that we'll stop relying on this
+	// action for launching tours
```

Comment:
> The bare minimum would be to cheat and still manually dispatch something in boot for the purpose of analytics, which something more sophisticated is conceived.

As long as it's equivalent to what we have now, I don't mind that :) I guess we can sort it out properly in another PR, before the themes tour goes live.